translations/frontend/en.json

```diff
@@ -1782,7 +1782,8 @@
                             "show_all": "Show all",
                             "show_disabled": "Show disabled entities",
                             "show_readonly": "Show read-only entities",
-                            "show_unavailable": "Show unavailable entities"
+                            "show_unavailable": "Show unavailable entities",
+							"show_with_area": "Show etities with area configured"
```

Comment:
Do not make changes to the translation files. Those are handled via "lokalise" project.

Reply:
Without this, I get an empty item in the menu
![image](https://user-images.githubusercontent.com/15268555/103369786-b7f87380-4ad3-11eb-9ed3-9722ca34b23f.png)

Reply:
The `script/develop` should be taking care of adding them to the EN translation file in GULP step `build-master-translation`.